### PR TITLE
Adding VPC peering between operations and data networks

### PR DESCRIPTION
**Pull request by blarghmatey**
- Added conditional importing of network resources defined in OLVPC component
- Added import of operations VPC
- Added peering between data and operations VPCs with associated routing rules

### CODE REVIEW

File: src/ol_infrastructure/lib/aws/ec2_helper.py [hunk 1]
```diff
@@ -18,7 +21,7 @@ def aws_regions() -> List[Text]:
 
 
 @lru_cache
-def availability_zones(region: Text) -> List[Text]:
+def availability_zones(region: Text) -> Tuple[Text]:
```
Comment (shaidar) [inline src/ol_infrastructure/lib/aws/ec2_helper.py:24]: Shouldn't `rtype` in doc string change as well?

**Comment by shaidar** [comment]
@blarghmatey Is the `pre-commit-config` intentional?

**Comment by blarghmatey** [comment]
Yes. I want it to be in the repo, but I can back it out and commit it directly to main if you prefer.

**Comment by shaidar** [comment]
👍

**Pull request merged**